CMakeLists.txt

```diff
@@ -16,16 +16,12 @@ else()
     message(STATUS "Found GLFW ${GLFW_PREFIX}")
 endif()
 
-list(APPEND GLFW_LDFLAGS
-    "-framework OpenGL"
-    "-framework Cocoa"
-    "-framework IOKit"
-    "-framework CoreFoundation"
-    "-framework CoreVideo")
-
 include_directories(include)
 
 add_executable(${EXECUTABLE_NAME} tests/main.cpp include/isect2d.h)
 
-target_link_libraries(${EXECUTABLE_NAME} ${GLFW_LDFLAGS})
+target_link_libraries(${EXECUTABLE_NAME} ${GLFW_STATIC_LIBRARIES})
```

Comment:
I'm having missing links on glfw3 using static libraries on osx, does this happen to you?